### PR TITLE
TextField: add max and min restrictions to set_scrollV too

### PR DESCRIPTION
**Pull request by junsred**
add restrictions to scrollV too

### CODE REVIEW

File: src/openfl/text/TextField.hx [hunk 1]
```diff
@@ -2812,6 +2812,9 @@ class TextField extends InteractiveObject
 	{
 		__updateLayout();
 
+		if (value > __textEngine.maxScrollV) value = __textEngine.maxScrollV;
+		if (value < 1) value = 1;
```
Comment (dimensionscape) [inline src/openfl/text/TextField.hx]: I think this(L#2816) might be handled somewhere else. I can't set scrollV to less than 1 in the latest git.

Comment (junsred) [reply src/openfl/text/TextField.hx]: Yeah nothing happens in latest git. What needs to happen is it should be set to 1.

Comment (dimensionscape) [reply src/openfl/text/TextField.hx]: Ah yeah that makes sense. Cool.

**Review by dimensionscape — CHANGES_REQUESTED** [review]
Ah it looks like this is partially handled in TextEngine but the `value>0` check prevents it. Just revert the initial commit and we're good.

**Comment by junsred** [comment]
@Dimensionscape after reverting my first commit, when i set the scrollV to a number that is higher than max scrollV it becomes max - 1 because of https://github.com/openfl/openfl/commit/f4fcd02408bfd4f0a9c2998bf507feadaddf1572 (Line: 1902) commit.

**Comment by junsred** [comment]
I don't see my first commit doing any harm.

**Comment by junsred** [comment]
Oh, I just realized that https://github.com/openfl/openfl/commit/f4fcd02408bfd4f0a9c2998bf507feadaddf1572 is fixed and the code is fine without first commit. I reverted it again. It should be ready for merge.

**Pull request merged**